### DISCUSSION TITLE
How to add CV as a badge?

### DISCUSSION THREAD

**Discussion by pinshuai** [alshedivat/al-folio#1879] [Q&A] [answered]
I would like to add a CV as a badge, similar to the Google Scholar badge but link to a PDF file. Is there a way to do this?

**Comment by george-gca** [comment, marked as answer]
Yes, you need to add you cv as an option in the [_config.yml](https://github.com/alshedivat/al-folio/blob/85f2590a2119ec57063845d628221155044ebea8/_config.yml#L67) file, as well as in the [_includes/social.html](https://github.com/alshedivat/al-folio/blob/master/_includes/social.html) file. Check how other socials are added this way.

**Reply by pinshuai** [reply]
Thanks George! For those who are interested, this is the way to add a link to a PDF.

```
            {%- if site.cv -%}
            <a href="{{ '/assets/pdf/' | append: site.cv | relative_url }}"><i class="fas fa-file-pdf""></i></a>
            {% endif %}
```
